Python/Product/VSInterpreters/InterpreterOptionsService.cs

```diff
@@ -226,7 +226,7 @@ sealed class InterpreterOptionsService : IInterpreterOptionsService {
             using (var key = Registry.CurrentUser.CreateSubKey(collection, true)) {
                 key.SetValue(LibraryPathKey, config.LibraryPath ?? string.Empty);
                 key.SetValue(ArchitectureKey, config.ArchitectureString);
-                key.SetValue(VersionKey, config.Version.ToString());
+                key.SetValue(VersionKey, config.Version?.ToString() ?? "2.7");
```

Comment:
No, but that's the default we've been using.  We should probably add a const somewhere and update all uses of it, but this was just fixing a crash I ran into.